.github/workflows/deploy.yml

```diff
@@ -0,0 +1,30 @@
+ame: deploy
+
+on: [push, pull_request]
+
+jobs:
+  deploy:
+    runs-on: "ubuntu-latest"
+    if: github.repository == 'quantumjot/BayesianTracker' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
Rather than doing this, could we not simply make it to build on `tag`?

Reply:
Probably - I was following the guidance from here:
https://github.com/pypa/gh-action-pypi-publish